File: sample/ScaffoldingSample.TypeScript/Models/Order.ts
import { Customer } from './Customer';
import { OrderDetail } from './OrderDetail';

export interface Order {
    orderId: number;
    customerId: string;
    orderDate: Date;
    shippedDate: Date;
    shipVia: number;
    freight: number;
    customer: Customer;
    orderDetails: OrderDetail[];
}

File: sample/ScaffoldingSample/Models/dbo/Order.cs
using System;
using System.Collections.Generic; // Comment

namespace ScaffoldingSample.Models.dbo
{ // Comment
    public partial class Order : EntityBase // My Handlebars Helper
    {
        public Order()
        {
            OrderDetails = new HashSet<OrderDetail>();
        }

        public int OrderId { get; set; } = default!;
        public string? CustomerId { get; set; }
        public DateTime? OrderDate { get; set; }
        public DateTime? ShippedDate { get; set; }
        public int? ShipVia { get; set; }
        public decimal? Freight { get; set; }

        public virtual Customer? Customer { get; set; }
        public virtual ICollection<OrderDetail> OrderDetails { get; set; } = default!;

        // My Handlebars Block Helper: True
        // My Handlebars Block Helper: False
    }
}
